tests/models/encoder_decoders/nlvr_semantic_parser_test.py

```diff
@@ -9,3 +14,35 @@ def setUp(self):
 
     def test_model_can_train_save_and_load(self):
         self.ensure_model_can_train_save_and_load(self.param_file)
+
+    def test_get_checklist_target(self):
+        # Creating a fake all_actions field where actions 0, 2 and 4 are terminal productions.
+        all_actions = [{"left": ("", True, {}), "right": ("", False, {})},
+                       {"left": ("", True, {}), "right": ("", True, {})},
+                       {"left": ("", True, {}), "right": ("", False, {})},
+                       {"left": ("", True, {}), "right": ("", True, {})},
+                       {"left": ("", True, {}), "right": ("", False, {})}]
+        # Of the actions above, those at indices 0 and 2 are on the agenda, and there are padding
+        # indices at the end.
+        test_agenda = Variable(torch.Tensor([[0], [4], [-1], [-1]]))
+        # pylint: disable=protected-access
+        target_checklist, relevant_actions = self.model._get_checklist_target(test_agenda,
+                                                                              all_actions)
+        checklist_data = target_checklist.data.numpy()
```

Comment:
Suggestion for brevity:

```python
numpy.testing.assert_almost_equal(target_checklist.data.numpy(), [[1], [0], [1]])
numpy.testing.assert_almost_equal(relevant_actions.data.numpy(), [[0], [2], [4]])
```

Just two lines, instead of six, and easier to read in a test - the relevant expected results are right next to each other.  Introducing new variables and having this large-looking code block makes me think the test is more complicated than it is.

Reply:
Done, thanks!